src/main/java/bibi/Ui.java

```diff
@@ -0,0 +1,140 @@
+package bibi;
+
+import bibi.task.Task;
+import bibi.task.TaskList;
+import java.util.Scanner;
+
```

Comment:
Should different groups of imports be separated by a whitespace and ordered differently? I noticed this issue in several other places.
e.g.
```
import java.util.Scanner;

import bibi.task.Task;
import bibi.task.TaskList;
```